### ISSUE TITLE
The profile configs are not compatible with the provided starter theme templates

### ISSUE THREAD

**Issue by h-archer** [AcroMedia/drupal-orange-project#22]
I installed a new Drupal instance with this profile using the command:

```bash
composer create-project acromedia/drupal-orange-project new_ecom_website --stability dev --no-interaction
```

Created a new database, then ran the gulp scss script with the following:

Goto theme directory:
```bash
cd new_ecom_website/web/themes/contrib/orange_ecom_starter
```
Install the npm packages and run the npm gulp script:
```bash
npm install && npm run gulp sass
```

Went to the browser by visiting this local website and ran the drupal install and selected this ecom orange profile.

All Good!

However, the profile uses Media fields for images. The profile configures all images as media fields (eg. for the sample product type and all the paragraphs), but the custom templates in the starter theme do not look for media image fields, they are looking for normal image fields. In other words, non of the images work within the templates for the product and all the paragraph twig files.

Basically the profile configurations for the fields do not work with the provided templates. It seems they are of different build versions, not in sync.

Can we get a sql dump of a working configured install that is compatible with the templates (and profile as a whole)?
